changelog/5975.deprecation.rst

```diff
@@ -0,0 +1,6 @@
+Deprecate using direct constructors for ``Nodes``.
+
+Instead they are new constructed via ``Node.from_parent``.
+
+This painful transitional mechanism enables us to detangle the very intensely
+entangled ``Node`` relationships.
```

Comment:
While I can feel you there it makes no sense to have the last paragraph in the changelog, does it?
If so, it should be reworded to be more neutral.

Reply:
define neutral there,

Reply:
Something without feelings (i.e. "painful").